sdk/formrecognizer/Azure.AI.FormRecognizer/src/Generated/Models/DataTableCell_internal.cs

```diff
@@ -30,6 +30,10 @@ internal DataTableCell_internal(int rowIndex, int columnIndex, string text, IEnu
             {
                 throw new ArgumentNullException(nameof(boundingBox));
             }
+            if (confidence == null)
```

Comment:
You can use `Argument.AssertNotNull(confidence, nameof(confidence))`

Reply:
This is a generated class though :(